src/Xamarin.Android.Build.Tasks/Utilities/ZipArchiveFileListBuilder.cs

```diff
@@ -0,0 +1,65 @@
+using System;
```

Comment:
For a new file, should we put `#nullable enable`? Might catch a random NRE.

Reply:
Definitely, added.